roles/rsyslog/vars/outputs/files/main.yml

```diff
@@ -62,25 +62,25 @@ __rsyslog_conf_files_output_rules:
 
               # Log anything (except mail) of level info or higher.
               # Don't log private authentication messages!
-              *.info;mail.none;authpriv.none;cron.none                {{ rsyslog_system_log_dir }}/messages
```

Comment:
Proposal for another PR - Could this inline string template be replaced by roles/rsyslog/templates/etc/rsyslog.conf.j2 ?

Reply:
sure, inline string templates would be better to have as template files (maybe avoid the `/etc`in the path according to https://github.com/oasis-roles/meta_standards/#ansible-best-practices "Avoid duplicating the remote full path in the role directory, however, as that creates unnecessary depth in the file tree for the role." )